pom.xml

```diff
@@ -499,6 +513,7 @@
     <project.httpclient.version>4.5.14</project.httpclient.version>
     <project.commons-codec.version>1.16.0</project.commons-codec.version>
     <project.oauth.version>1.35.0</project.oauth.version>
+    <project.auth.version>1.22.0</project.auth.version>
```

Comment:
We might discussed this in the past, but I'm a little surprised that this project is not dependent on auth library yet. We probably can not avoid it because we need to access the `Credentials` object for validation, but let's be more careful here as it may introduce dependency problem for customers. @suztomo Do you have any concerns regarding this new auth dependency?

Reply:
This does not create a circular dependency. Looks good to me.

![image](https://github.com/googleapis/google-api-java-client/assets/28604/7fcf764e-29c6-45fc-87fa-d325524541af)